cmd/lifecycle/exporter.go

```diff
@@ -197,12 +208,14 @@ func (e *exportCmd) export(group buildpack.Group, cacheStore lifecycle.Cache, an
 		AdditionalNames:    e.AdditionalTags,
 		AppDir:             e.AppDir,
 		DefaultProcessType: e.DefaultProcessType,
+		ExtendedDir:        e.ExtendedDir,
 		LauncherConfig:     launcherConfig(e.LauncherPath, e.LauncherSBOMDir),
 		LayersDir:          e.LayersDir,
 		OrigMetadata:       analyzedMD.Metadata,
 		Project:            projectMD,
 		RunImageRef:        runImageID,
-		Stack:              stackMD,
+		RunImageForExport:  runImageForExport,
+		Stack:              platform.StackMetadata{RunImage: runImageForExport}, // for backwards compat
```

Comment:
I'm actually not sure if we want this. This would allow newer images to run on older platforms. But maybe we should only populate this from stack.toml if it is present.

Reply:
I went back and read the code and `getRunImageForExport()` gets stack information from stack.toml for older platforms, so I think this is what we want.